content/200-concepts/150-data-platform/09-billing/07-payment-method-and-billing-information.mdx

```diff
@@ -87,9 +87,11 @@ Your billing information will be included in the next invoice issued for your us
 
 ## Business billing information
 
-### Add a business email
+### Add a billing email address
 
-You can add your company email to your monthly invoices.
+By default, you receive invoices on the email address of your GitHub account that you use to log in to the Prisma Data Platform.
+
+You can receive invoices to an additional email (such your company email).
```

Comment:
I think there's a small typo here: `such as your company email` or `like your company email`?

Reply:
Thanks, @martzoukos. This is now fixed.

I've also removed the sentence that mentions that this email replaces the GitHub in the invoices until we know more.